fetch.bs

```diff
@@ -3584,6 +3601,12 @@ with a <i>CORS flag</i> and <i>recursive flag</i>, run these steps:
    <a>queue a fetch-request-done task</a> for <var>request</var>.
   </ol>
 
+ <li><p>If <var>request</var>'s destination is "image" and <var>response</var>'s
+ <a for=response>header list</a> <a for="header list">contains</a>
+  `<code>Content-DPR</code>`, set <var>response</var>'s <a for=response>density</a> value to the
```

Comment:
This should be “*image* density", rather than just "density", to match up with the definition.

Reply:
good catch, thanks!